boardswarm-cli/src/ui.rs

```diff
@@ -209,11 +259,19 @@ pub async fn run_ui(
                         _ => (),
                     }
                 }
+                Some(new_state) = state_rx.recv() => {
```

Comment:
how about other states ? can we just pass them straight through to the UI ?

a match statememnt may be cleaner here